src/pages/ProcessBrowserAuth.vue

```diff
@@ -5,42 +5,21 @@
 <script>
```

Comment:
This endpoint is currently set as the clientEntry redirect uri and shows up in the login authorize params during login. Will these updates continue to serve that setup?

Reply:
This was only used by the popup login, which wasn't used. Server-side authentication goes to the /process-ssr-auth route, not this one. The checkSession calls appear to work by passing messages through the iframes, not relying on the redirect, but that's hard to confirm with looking at the code. I want to merge this into dev to test that out.